dapp/src/components/TransactionModal/ActiveUnstakingStep/SignMessageModal.tsx

```diff
@@ -106,7 +111,10 @@ export default function SignMessageModal() {
     (error: unknown) => {
       if (!sessionIdToPromise[sessionId.current].shouldOpenErrorModal) return
 
-      if (eip1193.didUserRejectRequest(error)) {
+      if (
+        eip1193.didUserRejectRequest(error) ||
+        ledgerLive.didUserRejectRequest(error)
```

Comment:
Can we combine these two with one common utils function?

Reply:
8b364a0